script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -11,7 +11,7 @@ An unwavering white silhouette with fluttering gold hair.
 [sha:597137f4150ff2c0aeec8b22c1a2678aea08420b]{
 -- Page 10012, Offset 17241.
 -- その姿は初めて見た時の姿と同じものだ。
-That figure is the same one I saw when I first laid eyes on her.
+The (abosolute) splitting image of when I first saw her. (Usage of "image" is quite different in this suggestion.) 
```

Comment:
I'd revert. I feel like **spitting** image usually means that two different things look alike, but this is hearkening back to an earlier image of the same person